.azure/pipelines/jobs/default-build.yml

```diff
@@ -193,6 +196,10 @@ jobs:
 
     - ${{ parameters.afterBuild }}
 
+    - ${{ if ne(parameters.agentOs, 'Windows') }}:
+      - script: df -h && du -h --threshold=50MB /home/vsts/work/
```

Comment:
Unfortunately macOS doesn't support `--threshold` (oops). Suggest using `-d 4` or similar on that `agentOS`.